pages/events/[id].tsx

```diff
@@ -0,0 +1,232 @@
+									</dd>
+								))}
+							</div>
+							<div className="sm:col-span-1">
+								<dt className="text-md font-medium text-gray-500">Stworzony</dt>
+								<dd className="text-md mt-1 text-gray-900">
+									{data.event?.createdAt}
+								</dd>
+							</div>
+						</dl>
+					</div>
+				) : (
+					<div className="mx-auto mt-6 max-w-5xl px-4 sm:px-6 lg:px-8">
+						<dl className="grid grid-cols-2 gap-x-4 gap-y-8">
+							<div className="sm:col-span-1">
+								<dt className="text-md font-medium text-gray-500">
+									Osób na imprezie
+								</dt>
+								<dd className="text-md mt-1 flex flex-row text-gray-900">
+									<UserIcon className="mt-1 mr-1 h-4 w-4" aria-hidden="true" />
+									{data.event?.participients}
+								</dd>
+							</div>
+							<div className="sm:col-span-1">
+								<dt className="text-md font-medium text-gray-500">Tix cena</dt>
+								<dd className="text-md mt-1 flex flex-row text-gray-900">
+									<Ticket className="mt-1 mr-1 h-4 w-4" aria-hidden="true" />
+									{data.event?.ticketPrice} PLN
+								</dd>
+							</div>
+						</dl>
+						<dl className="grid grid-cols-1 gap-x-4 gap-y-8 sm:grid-cols-2">
+							<div className="sm:col-span-1">
+								<nav
+									className="mt-6 h-full overflow-y-auto"
+									aria-label="Directory"
+								>
+									<dt className="text-md font-medium text-gray-500">
+										Przychody
+									</dt>
+									<div className="relative">
+										<ul role="list" className="relative z-0">
+											{data.event?.budgets.map((budget) => (
+												<li key={budget.id} className="bg-white">
+													<div
+														className={
+															budget.isIncome
+																? 'relative flex items-center space-x-3 border-b-2 border-gray-200 px-6 py-3'
+																: 'hidden'
+														}
+													>
+														<div className="min-w-0 flex-1">
+															<div className="focus:outline-none">
+																<span
+																	className="absolute inset-0"
+																	aria-hidden="true"
+																/>
+																<p className="text-sm font-medium text-gray-900">
+																	{budget.name}
+																</p>
+																<p className="truncate text-sm text-gray-500">
+																	{budget.amount} PLN
+																</p>
+															</div>
+														</div>
+													</div>
+												</li>
+											))}
+										</ul>
+									</div>
+								</nav>
+							</div>
+							<div className="sm:col-span-1">
+								<nav
+									className="mt-6 h-full overflow-y-auto"
+									aria-label="Directory"
+								>
+									<dt className="text-md font-medium text-gray-500">Wydatki</dt>
+									<div className="relative">
+										<ul role="list" className="relative z-0">
+											{data.event?.budgets.map((budget) => (
+												<li key={budget.id} className="bg-white">
+													<div
+														className={
+															budget.isIncome
```

Comment:
We can use here conditional rendering instead of hiding it with CSS. Also, a good approach would be to filter out data you want

Reply:
Conditional rendering between Budget/event